src/main/java/org/broadinstitute/hellbender/tools/copynumber/CollectReadCounts.java

```diff
@@ -158,19 +162,31 @@ public void onTraversalStart() {
 
         CopyNumberArgumentValidationUtils.validateIntervalArgumentCollection(intervalArgumentCollection);
 
-        logger.info("Initializing and validating intervals...");
-        final List<SimpleInterval> intervals = intervalArgumentCollection.getIntervals(sequenceDictionary);
-        intervalCachedOverlapDetector = new CachedOverlapDetector<>(intervals);
+        intervals = intervalArgumentCollection.getIntervals(sequenceDictionary);
+        intervalMultiset = HashMultiset.create(intervals.size());
 
-        //verify again that intervals do not overlap
-        Utils.validateArg(intervals.stream().noneMatch(i -> intervalCachedOverlapDetector.overlapDetector.getOverlaps(i).size() > 1),
-                "Input intervals may not be overlapping.");
+        final SAMFileHeader.SortOrder sortOrder = getHeaderForReads().getSortOrder();
+        isCoordinateSorted = sortOrder == SAMFileHeader.SortOrder.coordinate;
+        if (!isCoordinateSorted) {
```

Comment:
This and other checks might be overkill.  Not sure if they were inherited from older tools where sorting wasn't guaranteed?  We can get rid of them if anyone feels strongly.

Reply:
Sure, I agree.  Interval-overlap check is cheap and I'm OK with keeping it.